scripts/travis/.travis.yml

```diff
@@ -1,57 +1,41 @@
-sudo: true
 language: php
 dist: xenial
 
 # Adjust the version of PHP to match your production environment.
 # Making this version number greater than the production environment can have unintended consequences
 # including a non-functional prod environment.
-php:
-  - 7.2
-
-matrix:
-  fast_finish: true
+php: "7.2"
 
 env:
   global:
     - COMPOSER_BIN=$TRAVIS_BUILD_DIR/vendor/bin
     - BLT_DIR=$TRAVIS_BUILD_DIR/vendor/acquia/blt
-    - BUILD_DIR=$TRAVIS_BUILD_DIR
-
-jdk:
```

Comment:
Not necessary since BLT now uses the dmore behat chrome extension (direct Chrome debugging) rather than Selenium.